### ISSUE TITLE
Add code comments to all filters and hooks

### ISSUE THREAD

**Issue by arnaudbroes** [semperfiwebdesign/all-in-one-seo-pack#2072]
Ashish suggested that Ben and himself add code comments to all filters and hooks, on a per module basis. This both improves the code and makes it easier for Steve and I to write API documentation.

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

**Comment by arnaudbroes** [comment]
The sitemap and open graph modules should be done first since we added a few new filters there.

**Comment by michaeltorbert** [comment]
This is on hold, pending us getting out the PRs we already have. This seems like something likely to cause merge conflicts.
